README.md

```diff
@@ -67,74 +67,257 @@ For a more detailed example of how to use the contract client in conjunction
 with our CLI tools, XDR generation, and our XDR code generation tool: see the
 full contract [example](https://github.com/kochavalabs/full-contract-example).
 
+To run the sample code execute the following commands.
+
+Spawn a local Mazzaroth node.
+
+```bash
+docker run -p 8081:8081 kochavalabs/mazzaroth start standalone
+```
+
+Deploy the contract stated above.
+
+```bash
+mazzaroth-cli deploy deploy.json
+```
+
 ### Node-Client
 
 ```js
 import { NodeClient } from 'mazzaroth-js'
+import { fromPrivate } from 'mazzaroth-js/dist/crypto/ecc-ed25519'
```

Comment:
You can change this to use the public function:
AddressFromPrivate

https://github.com/kochavalabs/mazzaroth-js/blob/76e368b3e89c88ca7933acdee2094de7198943d6/src/index.js#L19

Reply:
Done. Thanks.